misc/formatter/modules/formatter-core/src/test/resources/actions/start/assert/start_action_2.bal

```diff
@@ -1 +1,3 @@
-future<int> f1 = @strand {thread: "any"} start multiply(1, 2);
+function foo() {
+    future<int> f1 = @strand {thread: "any"} start multiply(1, 2);
+}
```

Comment:
formattar changes also LGTM. as `future<int> f1 = @strand {thread: "any"} start multiply(1, 2);` is negative scenario at top level.

Reply:
Thanx @lochana-chathura